### ISSUE TITLE
Unclear feedback for maskable icons

### ISSUE THREAD

**Issue by PEConn** [GoogleChromeLabs/bubblewrap#167]
Run `npx @bubblewrap/cli init --manifest https://peconn.github.io/starters/manifest.json`.

After accepting the defaults for a few questions, you'll see:

```
? URL to an image that is at least 512x512px: https://peconn.github.io/starters/images/icons/pokeball-512.png
? URL to an image that is at least 512x512px to be used when generating maskable icons
```

It is unclear what happens if you leave this field blank. Furthermore if you do so, it gets reported as `undefined`, eg:

```
? URL to an image that is at least 512x512px: https://peconn.github.io/starters/images/icons/pokeball-512.png
? URL to an image that is at least 512x512px to be used when generating maskable icons undefined
```

We should make it clearer that there is a default option.

**Comment by yusufkandemir** [comment]
See #232

**Comment by yusufkandemir** [comment]
This issue can be closed as #232 got merged recently.

**Comment by andreban** [comment]
Closed via #232
